assignment_2_Script.js

```diff
@@ -0,0 +1,81 @@
+var dateCreation = (function iife() {
+  var currentModuleDate = null;
+  function instantiateDate() {
+    return new Date();
+  }
+  function setDate(date) {
+    if (date===undefined) {
+      currentModuleDate = instantiateDate();
+    }
+    else if (typeof date === 'number') {
+      currentModuleDate = new Date(date);
+    }
+    else if (date instanceof Date) {
+      currentModuleDate = new Date(date);
+    }
+  }
+  function getDate(requestedFormat) {
+    if (currentModuleDate === null) {
+      currentModuleDate = instantiateDate();
+    }
+    if (requestedFormat.format === 'formatted') {
+      return currentModuleDate.getMonth() + ' ' + currentModuleDate.getDate() +
+      ', ' + currentModuleDate.getFullYear();
+    }
+    else if (requestedFormat.format==='milliseconds' ||
+    requestedFormat===undefined) {
+      return currentModuleDate.getDate();
```

Comment:
`getDate` should return a millisecond value here, but `getDate` is going to return the date (of the month)